src/context.ts

```diff
@@ -6,8 +6,13 @@ import {
   useMemo,
   useRef,
 } from 'react'
-import { UseStore } from 'zustand'
-import { EqualityChecker, State, StateSelector } from './vanilla'
+import { EqualityChecker, UseStore } from 'zustand'
+import { State, StateSelector } from './vanilla'
+
+export interface UseStoreData<T extends State> {
```

Comment:
```suggestion
interface UseStoreData<T extends State> {
```
Yeah, let remove `export` for this time, because I'm not confident with the name.